lib/modules/manager/mint/extract.ts

```diff
@@ -0,0 +1,52 @@
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  const deps: PackageDependency[] = [];
+
+  content.split('\n').forEach((line) => {
```

Comment:
```suggestion
  content.split('\n').map(s => s.trim()).forEach((line) => {
```
use `for...of` loop instead `forEach`